charts/nitro/templates/servicemonitor.yaml

```diff
@@ -27,6 +27,10 @@ spec:
       tlsConfig: 
         {{- toYaml .Values.serviceMonitor.tlsConfig | nindent 8 }}
       {{- end }}
+      relabelings:
+        - targetLabel: chain
+          replacement: {{ .Values.chain.name }}
```

Comment:
Nitro can have the chain name set with `configmap.data.chain.name` Lets default to that if its set and allow it to be overridden.